op-node/node/conductor.go

```diff
@@ -0,0 +1,73 @@
+package node
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/ethereum-optimism/optimism/op-node/metrics"
+	"github.com/ethereum-optimism/optimism/op-node/rollup/driver"
+	"github.com/ethereum-optimism/optimism/op-service/dial"
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+	"github.com/ethereum-optimism/optimism/op-service/retry"
+	"github.com/ethereum/go-ethereum/log"
+
+	conductorRpc "github.com/ethereum-optimism/optimism/op-conductor/rpc"
+)
+
+// ConductorClient is a client for the op-conductor RPC service.
+type ConductorClient struct {
+	cfg       *Config
+	metrics   *metrics.Metrics
+	log       log.Logger
+	apiClient conductorRpc.API
+}
+
+// NoOpConductor is a no-op conductor that assumes this node is the leader sequencer.
+type NoOpConductor struct{}
+
+var _ driver.SequencerConductor = &NoOpConductor{}
+var _ driver.SequencerConductor = &ConductorClient{}
+
+// NewNoOpConductor returns a new conductor client for the op-conductor RPC service.
+func NewConductorClient(cfg *Config, log log.Logger, metrics *metrics.Metrics) *ConductorClient {
+	return &ConductorClient{cfg: cfg, metrics: metrics, log: log}
+}
+
+// Initialize initializes the conductor client.
+func (c *ConductorClient) Initialize() error {
+	conductorRpcClient, err := dial.DialRPCClientWithTimeout(context.Background(), time.Minute*1, c.log, c.cfg.ConductorEndpoint())
```

Comment:
There doesn't seem to be anything that closes this client again. I suspect the `ConductorClient` needs a `Close()` method that delegates through to the rpc client. Otherwise we may leave connections unclosed.

Reply:
Bump that `ConductorClient` still isn't closed at shutdown / closing it's resources.

Reply:
Added a Close call in driver.Stop()